bin/webpack-configs.js

```diff
@@ -775,11 +775,84 @@ const getStylingConfig = ( options = {} ) => {
 	};
 };
 
+const getInteractivityAPIConfig = ( options = {} ) => {
+	const { alias, resolvePlugins = [] } = options;
+	return {
+		entry: {
+			runtime: './assets/js/base/interactivity',
+		},
+		output: {
+			filename: 'wp-directives-[name].js',
+			path: path.resolve( __dirname, '../build/' ),
+		},
+		resolve: {
+			alias,
+			plugins: resolvePlugins,
+			extensions: [ '.js', '.ts', '.tsx' ],
+		},
+		plugins: [
+			...getSharedPlugins( {
+				bundleAnalyzerReportTitle: 'WP directives',
+			} ),
+			new ProgressBarPlugin(
+				getProgressBarPluginConfig( 'WP directives' )
+			),
+		],
+		optimization: {
+			runtimeChunk: {
+				name: 'vendors',
+			},
+			splitChunks: {
+				cacheGroups: {
+					vendors: {
+						test: /[\\/]node_modules[\\/]/,
+						name: 'vendors',
+						minSize: 0,
+						chunks: 'all',
+					},
+				},
+			},
+		},
+		module: {
+			rules: [
+				{
+					test: /\.(j|t)sx?$/,
+					exclude: /node_modules/,
+					use: [
+						{
+							loader: require.resolve( 'babel-loader' ),
+							options: {
+								cacheDirectory:
+									process.env.BABEL_CACHE_DIRECTORY || true,
+								babelrc: false,
+								configFile: false,
+								presets: [
+									[
+										'@babel/preset-react',
+										{
+											runtime: 'automatic',
+											importSource: 'preact',
+										},
+									],
+								],
+								plugins: [
+									'@babel/plugin-proposal-optional-chaining',
+								],
```

Comment:
```suggestion
								presets: [
									[
										'@wordpress/babel-preset-default',
										{
											modules: false,
											targets: {
												browsers: [
													'extends @wordpress/browserslist-config',
												],
											},
										},
									],
									[
										'@babel/preset-react',
										{
											runtime: 'automatic',
											importSource: 'preact',
										},
									],
								],
```

It seems that it is not necessary to install and use `@babel/plugin-proposal-optional-chaining`. We can use `@wordpress/babel-preset-default`.

Reply:
I have to say that I already tried, and `@wordpress/babel-preset-default` was messing up with Preact, adding `@wordpress/element` instead, so I opted to use that plugin instead of the preset you mentioned.

This is the part that replaces `createElement` with `@wordpress/element`:

https://github.com/WordPress/gutenberg/blob/trunk/packages/babel-preset-default/index.js#L80-L82